multy_core/src/golos/golos_account.cpp

```diff
@@ -41,6 +50,29 @@ uint32_t get_chain_index(BlockchainType blockchain_type)
         return CHAIN_INDEX_TEST;
     }
     return blockchain_type.blockchain;
+
+
+}
+
+// This method check that signature will be certain format
```

Comment:
Check that signature is in canonical format
For details please see: https://github.com/bitshares/bitshares1-core/issues/1129